bundles/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -215,7 +215,7 @@ Here are a few examples to unwrap a value from a complex response:
 |---------------------------------------------------------------------|-------------|-------------------------------------------|
 | `{device: {status: { temperature: 23.2 }}}`                         | JSONPATH    | `JSONPATH:$.device.status.temperature`    |
 | `<device><status><temperature>23.2</temperature></status></device>` | XPath       | `XPath:/device/status/temperature/text()` |
-| `THEVALUE:23.2°C`                                                   | REGEX       | `REGEX::(.*?)°`                           |
+| `THEVALUE:23.2°C`                                                   | REGEX       | `REGEX:(.*?)°`                           |
```

Comment:
IMO this is wrong. It'll return `THEVALUE:23.2` which is not the value alone. Either the one we have or `REGEX:THEVALUE:(.*?)°`.

Reply:
Hi J-N-K,
agree with your statement